pandas/io/excel.py

```diff
@@ -168,7 +168,20 @@ def get_writer(engine_name):
         raise ValueError("No Excel writer '%s'" % engine_name)
 
 
-@Appender(_read_excel_doc)
+def split_doc(doc_string, word_wrap=80):
```

Comment:
``Appender`` already calls ``dendent`` internally, I don't think you could add make it *also* call ``wrap`` [here](https://docs.python.org/2/library/textwrap.html) and it would just work. (the width could be an optional parameter to the ``Appender`` constructor).

But we will have to check some doc strings.